README.md

```diff
@@ -5,10 +5,26 @@
 
 ![Logo](https://raw.githubusercontent.com/opus111/FastAI.jl/master/fastai-julia-logo.png)
 
-This code is inspired by [fastai](https://github.com/fastai/fastai/blob/master/fastai/), but differs in implementation in several ways. Most importantly, the original Python code makes heavy use of side-effects where the `Learner` holds different state variables, and other objects access and modify them.
+FluxAI.jl is inspired by [fastai](https://github.com/fastai/fastai/blob/master/fastai/), and is a repository of best practices for Deep Learning with Flux.  Its goal is enable the development of state-of-the-art models, while freeing the developer from having to implement most of the components.  FastAI allows the design, training and delivery of models that compete with the best in class, using a few lines of code.  
 
-This has been replaced by a more functional design. The state is now transmitted via arguments to `Callbacks` which may then pass them on to `Metrics`.
+FastAI.jl contains thorough documentation, examples and tutorials, but does not contain the source of the core components.  It is an umbrella package combining the functionality specialized packages.  These packages include:
 
-*Note*: this is a package in-development. Expect breaking changes for the foreseeable future, but we want you to test out the package by following the documentation. Any contributions are welcome via PRs/issues.
+- Flux.jl: 100% pure-Julia Deep Learning stack. Provides lightweight abstractions on top of Julia's native GPU and AD support.
+
+- FluxTraining.jl: Enables eaily customized training loops, a large library of useful metrics, and many useful utilities
+
+- FluxONNX.jl: Support for reading and writing models in the ONNX file format.  ONNX enables Transfer Learning by enabling the import of very large pretrained models.
```

Comment:
Should we leave this as a comment for now until we have an official ONNX package?

Reply:
Any comments on this?

Reply:
Yeah, I agree